src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,121 @@
+import classNames from 'classnames';
+import { PersonLink } from '../PersonLink';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { useParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(res => setPeople(res))
+      .catch(() => setError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+  const { slug } = useParams();
+  const selectedPerson = slug;
```

Comment:
The variable `selectedPerson` is assigned the value of `slug` but is not used anywhere else in the code. Consider removing it if it's unnecessary or ensure it's used correctly if intended for a specific purpose.